changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Not yet release
+
+- Log client disconnected by same client_id
```

Comment:
I think it would be a bit clearer if it said something like : `- Log when a client is disconnected and multiple sessions are not allowed`.